src/types/io.ts

```diff
@@ -628,14 +628,24 @@ export interface AoARIOWrite extends AoARIORead {
     options?: WriteOptions,
   ): Promise<AoMessageResult>;
   // END OF GATEWAY SPECIFIC INTERACTIONS
+  // @deprecated - use `buyArNSName` instead
```

Comment:
tsdoc syntax will provide IDE hinting and ~~strikethrough~~

```suggestion
  /** @deprecated - use `buyArNSName` instead */
```

Reply:
[30c2e51](https://github.com/ar-io/ar-io-sdk/pull/330/commits/30c2e5111fe8d34d96ca9e002074ca24f37edd57)